src/core/ColorPicker/ColorSwatch.tsx

```diff
@@ -22,33 +23,34 @@ export type ColorProps = {
    * User friendly color name.
    */
   name?: string;
-  /**
-   * Tooltip ref prop.
-   */
-  tooltipRefProp?: React.ComponentPropsWithRef<'span'>;
 } & CommonProps &
   React.ComponentPropsWithRef<'span'>;
 
-export const Color = (props: ColorProps) => {
-  const { color, style, onClick, isActive, tooltipRefProp, ...rest } = props;
+export const Color = React.forwardRef<HTMLSpanElement, ColorProps>(
+  (props, ref) => {
+    const { color, style, onClick, isActive, ...rest } = props;
+
+    useTheme();
 
-  useTheme();
+    const spanRef = React.useRef<HTMLSpanElement>(null);
```

Comment:
No need to create `spanRef` here as you are not using it in the component anywhere.
Just take the `ref` which you have forwarded and add it to the span (no need to merge those refs then).

Reply:
I removed it